packages/bson-transpilers/test/yaml/driver-syntax.yaml

```diff
@@ -177,6 +206,27 @@ tests:
                     result = client.database['collection'].find({
                       'x' => 0
                     })
+                swift: |-
+                    // Requires the MongoSwift package
```

Comment:
Similar to the `throws` problem, top level `async` isn't supported yet, so a user might get compilation issues if they just throw this in a `.swift` file. Should we add some scaffolding here to handle that? If so, what would that be?

Reply:
sounds good to me 👍